app/src/main/res/values/strings.xml

```diff
@@ -1,41 +1,41 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <string name="app_name">Checkin</string>
     <string name="account_type">com.checkin.accounts</string>
-
+    
     <string name="app_old_version_button_update">Update app</string>
     <string name="app_old_version_dialog_title">Update your app</string>
     <string name="app_old_version_dialog_message">Your app needs to be updated\nfrom the play store to use it.</string>
-
+    
     <string name="navigation_drawer_open">Open navigation drawer</string>
     <string name="navigation_drawer_close">Close navigation drawer</string>
     <string name="tables_drawer_open">Open tables drawer</string>
     <string name="tables_drawer_close">Close tables drawer</string>
     <string name="appbar_scrolling_view_behavior" translatable="false">com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior</string>
-
+    
     <string name="action_settings">Settings</string>
-
+    
     <string name="action_search">Search</string>
     <string name="action_finish">Finish</string>
     <string name="action_filter">Filter Items</string>
     <string name="label_about">About Us</string>
-
+    
     <!--for bottomnav bar-->
-
+    
     <!-- value for the phone mode btn_menu -->
     <string name="bbn_phone_view_behavior" translatable="false" tools:ignore="UnusedResources">it.sephiroth.android.library.bottomnavigation.BottomBehavior</string>
-
+    
     <!-- behavior for the tablet mode btn_menu -->
     <string name="bbn_tablet_view_behavior" translatable="false" tools:ignore="UnusedResources">it.sephiroth.android.library.bottomnavigation.TabletBehavior</string>
-
+    
     <!-- this is the default value -->
     <string name="bbn_default_view_behavior" translatable="false" tools:ignore="UnusedResources">it.sephiroth.android.library.bottomnavigation.BottomBehavior</string>
-
+    
     <!-- default badge provider -->
     <string name="bbn_badgeProvider" translatable="false" tools:ignore="UnusedResources">it.sephiroth.android.library.bottomnavigation.BadgeProvider</string>
-
+    
     <!-- default floating action button behavior -->
     <string name="bbn_fab_default_behavior" translatable="false" tools:ignore="UnusedResources">it.sephiroth.android.library.bottomnavigation.FloatingActionButtonBehavior</string>
-
+    
```

Comment:
What kind of formatting have you enabled in AndroidStudio? Why will it explicitly add 4 spaces in blank lines?

Reply:
Actually i have no idea, Thats why the previous time i asked for the kind of indentation details you use